docs/content/odh/trino/trino_superset_user_guide.md

```diff
@@ -94,11 +94,11 @@ https://trino.operate-first.cloud](https://trino.operate-first.cloud).
 
 After logging in you should see a landing page which displays the number of queries running, active workers, queries executed etc:
 
-<img src="assets/images/trino-landing-page.png" width="500">
+![trino-landing-page](assets/images/trino-landing-page.png)
 
 You can click on a particular query ID to view its logs like so:
 
-<img src="assets/images/trino-query-page.png" width="500">
+![trino-query-page]assets/images/trino-query-page.png)
```

Comment:
doesn't work, I think you are missing a `(`

Reply:
@HumairAK thanks. fixed it